docs/pages/eas-update/publish.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: How to publish an update
+sidebar_title: Publish an update
+description: Learn how to publish an update to a specific branch with EAS Update.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { LayersTwo02Icon } from '@expo/styleguide-icons';
+
+Publishing an update allows:
+- Fixing bugs and quickly updating non-native parts of a project instead of creating a new build
+- [Sharing a preview version of an app](/review/overview/) using internal distribution
```

Comment:
@brentvatne, as per your recent feedback, I've added a link to Review > Overview that mentions sharing a preview version.